### PR TITLE
[DTO54][Thanh] Integrated AWS Cognito for authentication

### PR DESCRIPTION
**Pull request by megastar24577**
## What does this PR solve?
Using third party AWS Cognito for login with OAuth2

### CODE REVIEW

**Review by buigiaanfb1 — COMMENTED** [review]
Congrats on your PR, @megastar24577 , 
I have some minor comments, please take a look:

File: src/hooks/useAuth.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { useContext } from 'react';
-import { AuthContext } from '../contexts/JWTContext';
-// import { AuthContext } from '../contexts/AwsCognitoContext';
+// import { AuthContext } from '../contexts/JWTContext';
+import { AuthContext } from '../contexts/AwsCognitoContext';
 // import { AuthContext } from '../contexts/Auth0Context';
 // import { AuthContext } from '../contexts/FirebaseContext';
```
Comment (buigiaanfb1) [inline src/hooks/useAuth.ts:5]: Better if we delete these comments!

File: src/index.tsx [hunk 2]
```diff
@@ -49,8 +49,8 @@ import LoadingScreen from './components/LoadingScreen';
 // Check our docs
 // https://docs-minimals.vercel.app/authentication/ts-version
 
-import { AuthProvider } from './contexts/JWTContext';
-// import { AuthProvider } from './contexts/AwsCognitoContext';
+/* import { AuthProvider } from './contexts/JWTContext'; */
+import { AuthProvider } from './contexts/AwsCognitoContext';
 // import { AuthProvider } from './contexts/Auth0Context';
 // import { AuthProvider } from './contexts/FirebaseContext';
```
Comment (buigiaanfb1) [inline src/index.tsx:55]: Same!

**Review by buigiaanfb1 — APPROVED** [review]
LGTM! Thanks, @megastar24577

File: src/index.tsx (hunk 2 above)
Comment (megastar24577) [reply src/index.tsx:55]: Temporary leave it for later maybe use

**Pull request merged**